numpy/testing/_private/utils.py

```diff
@@ -2388,6 +2388,22 @@ def requires_memory(free_bytes):
     """Decorator to skip a test if not enough memory is available"""
     import pytest
 
+    def decorator(func):
+        @wraps(func)
+        def wrapper(*a, **kw):
+            msg = check_free_memory(free_bytes)
+            if msg is not None:
+                pytest.skip(msg)
+            return func(*a, **kw)
+        return wrapper
```

Comment:
Be good to put a blank line after the function decorator.

Reply:
ok